docs/examples/guides/estimators_guide.py

```diff
@@ -22,7 +33,7 @@ def __call__(
 from functools import partial
 
 import numpy as np
-from orquestra.integrations.qulacs import QulacsSimulator
+from orquestra.integrations.qulacs.simulator import QulacsSimulator
```

Comment:
This extra `simulator` shouldn't be necessary here.
Normally I wouldn't care, but this is for docs, so e we need to give people good example on how to use our imports :)